### PR TITLE
OGM-1265. Add new operations to native CLI in MongoDB

### PR DESCRIPTION
**Pull request by schernolyas**
The PR contains part of operations. See list below.

1. Add support of operation 'insertOne'
2. Add support of operation 'insertMany'
3. Add support of operation 'deleteOne'

### CODE REVIEW

**Comment by schernolyas** [comment]
The PR contains:
1) https://hibernate.atlassian.net/browse/OGM-1310
2) https://hibernate.atlassian.net/browse/OGM-1311 
2) https://hibernate.atlassian.net/browse/OGM-1313

**Comment by DavideD** [comment]
I'm starting the review

File: mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/query/nativequery/MongoDBSessionCLIQueryTest.java [hunk 1]
```diff
@@ -236,6 +236,70 @@ public void testInsertThenRemove() throws Exception {
...
+			// And check that they are gone.
+			nativeQuery = "db." + OscarWildePoem.TABLE_NAME + ".findOne({ '_id': { '$numberLong': '11' } })";
+			query = session.createNativeQuery( nativeQuery ).addEntity( OscarWildePoem.class );
+			result = query.list();
+			assertThat( result.size() ).isEqualTo( 0 );
+
+			nativeQuery = "db." + OscarWildePoem.TABLE_NAME + ".findOne({ '_id': { '$numberLong': '12' } })";
+			query = session.createNativeQuery( nativeQuery ).addEntity( OscarWildePoem.class );
+			result = query.list();
+			assertThat( result.size() ).isEqualTo( 0 );
```
Comment (DavideD) [inline mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/query/nativequery/MongoDBSessionCLIQueryTest.java]: I'm pretty sure that all the cleanup code could become something like: `delete( session, 11, 12 )`.

Where `delete` is the existing method we already call in `@After tearDown()` but change to the following signature:

    private void delete(final Session session, final Object... ids) {
	...
   }

**Comment by DavideD** [comment]
I'm going to close this PR to have less notification, feel free to send a new one when you have applied the remarks

**Pull request closed without merging**